src/modules/deployment/reducer.spec.ts

```diff
@@ -65,3 +74,44 @@ describe('when DEPLOY_TO_WORLD_FAILURE action is dispatched', () => {
     expect(deploymentReducer(INITIAL_STATE, deployToWorldFailureAction)).toEqual(expect.objectContaining({ error: 'error' }))
   })
 })
+
+describe('when FETCH_WORLD_DEPLOYMENTS_REQUEST action is dispatched', () => {
+  it('should add loading to fetch world deployments', () => {
+    const fetchWorldDeploymentsRequestAction = fetchWorldDeploymentsRequest(mockWorlds)
+    expect(deploymentReducer(INITIAL_STATE, fetchWorldDeploymentsRequestAction)).toEqual(
+      expect.objectContaining({ loading: [fetchWorldDeploymentsRequestAction] })
+    )
+  })
+})
+
+describe('when FETCH_WORLD_DEPLOYMENTS_SUCCESS action is dispatched', () => {
+  it('should remove loading to fetch world deployments', () => {
+    const fetchWorldDeploymentsSuccessAction = fetchWorldDeploymentsSuccess(mockWorlds, [{}] as Deployment[])
+    expect(deploymentReducer(INITIAL_STATE, fetchWorldDeploymentsSuccessAction)).toEqual(expect.objectContaining({ loading: [] }))
```

Comment:
isn't initial state already loading: [] ? maybe you can test that is was there before and it isn't there anymore just in case

Reply:
Thanks for the review! good catch, this test it's nonsense because the initial state is already `loading:[]` 😬